docs/data/sdks/react-native-sdk.md

```diff
@@ -392,28 +392,35 @@ setOptOut(false);
 
 ### Optional tracking
 
-By default, the SDK tracks some properties automatically. You can override this behavior by passing an object called `trackingOptions` when initializing the SDK, setting the appropriate options to false.
+By default, the SDK tracks these properties automatically. You can override this behavior by passing a configuration called `trackingOptions` when initializing the SDK, setting the appropriate options to false.
 
 | Tracking Options | Default |
 | --- | --- |
-| `city` | `true` |
-| `country` | `true` |
+| `adid` | `true` |
 | `carrier` | `true` |
 | `deviceManufacturer` | `true` |
 | `deviceModel` | `true` |
-| `dma` | `true` |
 | `ipAddress` | `true` |
 | `language` | `true` |
 | `osName` | `true` |
 | `osVersion` | `true` |
 | `platform` | `true` |
-| `region` | `true` |
-| `versionName` | `true` |
```

Comment:
Do we want to keep `versionName`? Mentioned in https://github.com/amplitude/Amplitude-TypeScript/issues/179

Reply:
it's actually not being used right now so behavior does not change regardless of value. though i see that it potentially be used somewhere. what do you think?

Reply:
Got it, then think we can remove, thanks for checking this!